cucumber/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem "calabash-cucumber"
-gem "run_loop", :github => "calabash/run_loop", :branch => "develop"
+gem "run_loop", :github => "sergey-plevako-badoo/run_loop", :branch => "add_setting_server_port"
```

Comment:
Rollback this change.  I have merged the your run-loop PR.

Reply:
fixed